reposerver/repository/repository.go

```diff
@@ -1270,8 +1270,11 @@ func runConfigManagementPlugin(appPath, repoRoot string, envVars *v1alpha1.Env,
 	return kube.SplitYAML([]byte(out))
 }
 
-func getPluginEnvs(envVars *v1alpha1.Env, q *apiclient.ManifestRequest, creds git.Creds) ([]string, error) {
-	env := append(os.Environ(), envVars.Environ()...)
+func getPluginEnvs(envVars *v1alpha1.Env, q *apiclient.ManifestRequest, creds git.Creds, remote bool) ([]string, error) {
```

Comment:
Can you add a comment to describe what remote does? And, if you have time, a unit test to illustrate the difference?

Reply:
I have added a comment. Testing the feature at the level of exported functions would require mocking the cmp server which does not look like an easy task. I do not have the time for that.